admin_scripts/rotate_display.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    rotate_display.sh [normal/right/left]
+#%
+#% DESCRIPTION
+#%    This script looks for all displays connected and rotates the first one from the list.
+#%
+#%    It takes one mandatory parameter. The direction of the rotation.
+#%    Based on xrandr. From xrandr manual:
+#%    "Rotation can be one of 'normal', 'left', 'right' or 'inverted'. This causes the output contents to be
+#%    rotated in the specified direction. 'right' specifies a clockwise rotation of the picture and  'left'
+#%    specifies a counter-clockwise rotation."
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         rotate_display.sh (magenta.dk) 1.0.0
+#-    author          Danni Als
+#-    copyright       Copyright 2019, Magenta ApS
+#-    license         GNU General Public License
+#-    email           danni@magenta-aps.dk
+#-
+#================================================================
+#  HISTORY
+#     2019/11/12 : af : Script created
+#     2019/11/14 : af : Changed tactics. Instead of trying to connect to the X-server from root, we now rotate screen upon user login.
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+set -e
+
+if [ "$1" != "normal" -a "$1" != "right" -a "$1" != "left" ]; then
+    echo "Wrong rotation command given: $1"
+    exit -1
+fi
+
+AUTOSTART_FOLDER=/home/.skjult/.config/autostart/
+
+if [ ! -d $AUTOSTART_FOLDER ]; then
+    mkdir $AUTOSTART_FOLDER
+    echo "$AUTOSTART_FOLDER created..."
+fi
```

Comment:
One simpler option here is the `mkdir -p` command, which also makes all the folders necessary to make the destination folder and which _doesn't_ fail if the folder already exists.

Reply:
Yes but then we cant see if it was created or not unless the command gives us output of what happened.